docs/warning-codes.md

```diff
@@ -88,4 +88,6 @@
 ### 43
 `Agent not configured properly.`
 ### 44
-`Invalid object passed to generic event aggregate. Missing "eventType".`
\ No newline at end of file
+`Invalid object passed to generic event aggregate. Missing "eventType".`
+### 45
```

Comment:
i had added this with a `warn`, but then thought the warn was potentially too noisy. I'm just going to leave the code in since it can be used later for whatever and it has no impact on the agent.